docs-v2/mint.json

```
@@ -3,7 +3,8 @@
     "name": "Nango Docs",
     "logo": {
         "light": "/logo/light.svg",
-        "dark": "/logo/dark.svg"
+        "dark": "/logo/dark.svg",
+        "href": "https://nango.dev"
```

```suggestion
        "href": "https://www.nango.dev"
```
neatpick: avoid a redirect since `https://nango.dev` is redirected to `https://www.nango.dev`